src/execution/sql/functions/string_functions.cpp

```diff
@@ -447,4 +454,55 @@ void StringFunctions::Chr(StringVal *result, exec::ExecutionContext *ctx, const
     }
   }
 }
+
+void StringFunctions::Md5(StringVal *result, exec::ExecutionContext *ctx, const StringVal &str) {
+  if (str.is_null_) {
+    *result = StringVal::Null();
+    return;
+  }
+  char new_str[str.GetLength()];
+  memcpy(new_str, str.GetContent(), str.GetLength());
+  MD5_CTX md5ctx;
+  MD5Init(&md5ctx);
+  MD5Update(&md5ctx, reinterpret_cast<unsigned char *>(new_str), str.GetLength());
+  unsigned char md5sum[16];
+  MD5Final(&md5ctx, md5sum);
```

Comment:
I'm not sold on this library. I vote we leave this function out of the PR until we can vet it properly.

Reply:
I think that's reasonable, I don't know anything about the library either tbh because it came from the original PR.